src/sentry/static/sentry/app/utils.tsx

```diff
@@ -282,3 +282,8 @@ export function generateQueryWithTag(
 }
 
 export const isFunction = (value: any): value is Function => typeof value === 'function';
+
+// NOTE: only escapes a " if it's not already escaped
+export function escapeDoubleQuotes(str) {
+  return str.replace(/\\([\s\S])|(")/g, '\\$1$2');
+}
```

Comment:
Should this be part of the queryString module that includes the tokenize/stringify search?

Reply:
I'll explore this.

Reply:
Addresseed in https://github.com/getsentry/sentry/pull/17763/commits/1557eac750db10715f6f6fec3e0b9ec3a3fa4f96